include/constants/battle.h

```diff
@@ -669,25 +669,45 @@ enum FaintedActions
     FAINTED_ACTIONS_MAX_CASE,
 };
 
-// Constants for B_VAR_STARTING_STATUS
-// Timer value controlled by B_VAR_STARTING_STATUS_TIMER
+//  Enum,                                         fieldName,           Type, max value
+#define STARTING_STATUS_DEFINITIONS(F) \
+    F(STARTING_STATUS_ELECTRIC_TERRAIN,               electricTerrain,            (u32, 1)) /* Electric Terrain (Permanent) */     \
+    F(STARTING_STATUS_ELECTRIC_TERRAIN_TEMPORARY,     electricTerrainTemporary,   (u32, 1)) /* Electric Terrain (5 turns) */       \
```

Comment:
```suggestion
    F(STARTING_STATUS_ELECTRIC_TERRAIN_TEMPORARY,     electricTerrainTemporary,   (u32, 1)) /* Electric Terrain Temporary (5 turns) */       \
```
could you add `Temporary` everywhere for the 5 turn stuff?